src/translations/en.json

```diff
@@ -1898,6 +1898,7 @@
           },
           "edit_card": {
             "header": "Card Configuration",
```

Comment:
Is this used somewhere else?

Reply:
Yes a few lines down in the `:` expression